Demo/EarlGreyExample/earlgrey_gem.yml

```diff
@@ -0,0 +1,4 @@
+EarlGreyExampleSwiftTests:
```

Comment:
this is cool, didn't know about this setting, could you also update the doc?

Reply:
updated the readme